src/main/java/com/mappy/fpm/batches/tomtom/shapefiles/RailwayShapefile.java

```diff
@@ -28,7 +28,8 @@ public void serialize(GeometrySerializer serializer, Feature feature) {
         Map<String, String> tags = Maps.newHashMap();
         tags.put("ref:tomtom", String.valueOf(feature.getLong("ID")));
         tags.put("railway", "rail");
-        tags.putAll(level(feature));
+        Boolean isReversed = "TF".equals(feature.getString("ONEWAY"));
```

Comment:
The railway does not have associated restrictions, so we only use the oneway field